tests/TestCase/Validation/ValidationTest.php

```diff
@@ -2122,6 +2122,22 @@ public function testIpv6(): void
         $this->assertFalse(Validation::ip('255.255.255.255', 'ipv6'), 'IPv4 is not valid IPv6');
     }
 
+    /**
+     * Tests ipOrSubnet().
+     */
+    public function testIpOrSubnet(): void
+    {
+        $this->assertTrue(Validation::ipOrSubnet('192.168.1.0'));
```

Comment:
Should this method name be `ipSubnet`? Or `cidrRange`?

Reply:
Well, ipSubnet sounds like like it would do subnets, but it accepts ip or range.
and cidrRange is not a term I am familar with - or others might be.

Reply:
ipOrRange() might be more speaking.